### DISCUSSION TITLE
TuYa TS0207 Repeater is not repeating

### DISCUSSION THREAD

**Discussion by NeWoKiKi** [Koenkk/zigbee2mqtt#17788] [Devices]
Hi,

I bought a TuYa TS0207 Repeater and added it in Zigbee2MQTT without any problem but it's not doing its job.
If I check in Schema section, I can see that my repeater isn't repeating anything (there are some devices closer to the repeater than coordinator).
Do you know why ? Is there a way to make it work normally ?

Schema :
![image](https://github.com/Koenkk/zigbee2mqtt/assets/69678428/aa610880-09fb-430d-bf66-7ec666888b09)

**Comment by lux73** [comment]
especially Aqara Devices are known to not switch to other/better Routes/Routers automaticially

afaik you have to delete them from z2m manually and re-pair them near the new devices - but even that's not guaranteed that they will connect with the new router ^^

**Comment by NeWoKiKi** [comment]
Hi,

After deleting one of my sensor and adding it again near repeater, my device is now connected to the repeater.
Thanks for this "work around", hoping it will stay like this :)

![image](https://github.com/Koenkk/zigbee2mqtt/assets/69678428/cf0110b2-3297-411b-afaf-f4837a217c05)
